src/components/ui/sidebar/nav-main.tsx

```diff
@@ -120,7 +122,7 @@ export function NavMain({ links }: { links: NavigationLink[] }) {
                       exactActiveClass="bg-white text-green-700 shadow-sm"
                     >
                       {link.icon ? (
-                        link.icon
+                        <CareIcon icon={link.icon as IconName} />
```

Comment:
let's not do type-casting!

the right solution here is to fix the root cause!

see what was done here: https://github.com/ohcnetwork/care_fe/commit/0c135565b93ffa8ebec3b1c2e413ce74dfe27832

this file was missed out accidentally I believe.